### ISSUE TITLE
Farming with New NFT plots only Get 4 SIX Rewards

### ISSUE THREAD

**Issue by wrjmhb21** [dream80/lucky-blockchain#4]
I am using the new NFT plot for solo farming, not pool farming, but only got 4 SIX rewards, where is the other 2 SIX, and how can I get it?

**Comment by dream80** [comment]
Yes, NFT can only get 2/3, and most forks can only get 1/8. It seems that there is a project that can help get another part back, but I'm not sure yet.
